content/about/about_people.rst

```diff
@@ -485,6 +485,28 @@ https://github.com/athai
 
   <br/>
 
+`Cayleigh Thalmann`_
+--------------------
+
+.. image:: /images/cthalmann.jpg
```

Comment:
I spelled your name wrong on the picture (lol). You'll either need to change the picture's name in the file I sent you or change it here. Mea culpa.